torch/_inductor/compile_fx.py

```diff
@@ -231,7 +231,8 @@ def is_aligned(storage_offset, dtype):
     check_inputs = [
         i
         for i in range(len(inputs))
-        if (
+        if not torch.is_tensor(inputs[i])
+        or (
```

Comment:
Now, non-tensor inputs can appear here. This check is because of `storage_offset` and `device.type` below.